iSH.xcconfig

```diff
@@ -1,5 +1,5 @@
 // Change this to change all the bundle IDs and app groups
-ROOT_BUNDLE_IDENTIFIER = app.ish.iSH
+ROOT_BUNDLE_IDENTIFIER = app.ish.mkeiSH
```

Comment:
Was this swap intentional?

Reply:
Yes.  Though I should have probably excluded iSH.xcconfig and project.pbxproj from the pull request.  I'm new to git and still learning.

Reply:
Like I said remove since bundle identifier didn’t change.

Reply:
Will do.  Working on that now.  Hopefully this will also keep me from having to manually change all this stuff everytime I pull updates from the mainline master as well.